configs/benchmarks/classification/_base_/models/vit_base_16.py

```diff
@@ -0,0 +1,18 @@
+model = dict(
```

Comment:
I think using the config file name format like `vit-base-p16` is better, the same name as mmcls.

Reply:
Thank you for your valuable feedbacks. Your suggestion is reasonable and I will fix it.